src/style.css

```
@@ -1 +1,73 @@
-/* Styles go here */
+:root {
+  --color-hover: #00acdc;
+  --font-size-main: 12px;
+  --color-main: #000;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 100%;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
```

it's better to use :not(:last-child)
```suggestion
.nav__item:not(:last-child) {
  margin-right: 20px;
}
```